catalog/lib/extensions/color_extensions.dart

```diff
@@ -0,0 +1,61 @@
+import 'dart:math';
+import 'package:flutter/material.dart';
+
+extension ColorExtension on Color {
+  Color _getShade(int shade) => this is MaterialColor
+      ? (this as MaterialColor)[shade]!
+      : _generateMaterialColor();
+
+  MaterialColor _generateMaterialColor() => MaterialColor(value, {
+        50: _tintColor(0.9),
```

Comment:
shouldn't this be shade instead of tint? whats the difference?

Reply:
The difference it is the way of calculate tint and shade